cmd/kueuectl/app/list/list_workload.go

```diff
@@ -188,14 +232,45 @@ func (o *WorkloadOptions) Run(ctx context.Context) error {
 		namespace = ""
 	}
 
+	var jobUID types.UID
+	var jobUIDLabelSelector string
+	if o.UnstructuredObjectsStream != nil {
```

Comment:
I still find this unreadable.

It would be better if WorkloadOptions contains only the job as `unstructured.Unstructured`, calculated during `Complete`.

Otherwise, someone reading the `Run` function sees `UnstructuredObjectsStream` and wonders what could it be. The only way for them to tell is to go into the `Complete` function.

Reply:
Done.